siva/transactioner_test.go

```diff
@@ -25,7 +25,7 @@ func setupTranstactioner(t *testing.T) *transactioner {
 	require.True(ok)
 	require.Equal(loc, lr)
 
-	txer := newTransactioner(loc, locReg)
+	txer := newTransactioner(loc, locReg, 100*time.Millisecond)
```

Comment:
I think it should be better if `setupTransactioner` takes in a parameter to set the timeout, so in `TestTransactioner` we give it the default value and for `TestTransactioner_Timeout` we set `100*time.Millisecond`